src/Murder.Editor/Utilities/OperatingSystemHelpers.cs

```diff
@@ -145,7 +169,7 @@ private static bool SetTextForOsx(string text)
         return null;
     }
 
-    public const string AppKit = "/System/Library/Frameworks/AppKit.framework/AppKit";
+    public const string AppKit = "/System/Library/Frameworks/AppKit.framework/Versions/Current/AppKit";
```

Comment:
did they change the path for this?

Reply:
I'm not sure, my version of macOS couldn't find this dynamic library. The "/System/Library/Frameworks/AppKit. framework/AppKit" location is just an empty soft link